docs/source/references/schema.json

```diff
@@ -3714,7 +3991,7 @@
       },
       {
         "name": "payload",
-        "type": "ClientQueryPayload"
+        "type": "QueryRequestWithAuthority"
```

Comment:
```suggestion
        "name": "request",
        "type": "QueryRequestWithAuthority"
```

Reply:
I think the `payload` is fitting, as it signifies the part that is signed (signature payload). Not sure that `request` is better, especially considering `QueryRequestWithAuthority` itself also has a `request` field..